swift/internal/debugging.bzl

```diff
@@ -61,9 +62,14 @@ def ensure_swiftmodule_is_embedded(
         # For ELF-format binaries, we need to invoke a Swift modulewrap action
         # to wrap the .swiftmodule file in a .o file that gets propagated to the
         # linker.
+        add_target_name_to_output_path = is_feature_enabled(
+            feature_configuration = feature_configuration,
+            feature_name = SWIFT_FEATURE_ADD_TARGET_NAME_TO_OUTPUT,
+        )
         modulewrap_obj = derived_files.modulewrap_object(
```

Comment:
The comment above (`# For ELF-format binaries, we need to invoke a Swift modulewrap action...`) is now missplaced. I think better:
```
        add_target_name_to_output_path = is_feature_enabled(
            feature_configuration = feature_configuration,
            feature_name = SWIFT_FEATURE_ADD_TARGET_NAME_TO_OUTPUT,
        )
        # For ELF-format binaries, we need to invoke a Swift modulewrap action
        # to wrap the .swiftmodule file in a .o file that gets propagated to the
        # linker.
        modulewrap_obj = derived_files.modulewrap_object(
```

Reply:
Fixed